mm.c

```diff
@@ -381,10 +380,24 @@ int mm_init(void)
 {
     // Initialize the free list
     for (int i = 0; i <= MAX_POWER; i++) {
-				SET_FREE_LIST_PTR(i, NULL);
+	    SET_FREE_LIST_PTR(i, NULL);
     }
 
+    char *heap_start; // Pointer to begining of heap 
```

Comment:
There is already a global variable called `heap_ptr` that you should set instead.